.github/ISSUE_TEMPLATE/internal_ux.yml

```diff
@@ -0,0 +1,53 @@
+name: (Internal) UX Template
+description: Intended to help ux create internal flows.
+title: "[UX]: "
+labels: ["kind/ux"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A introductory description of the task
+    validations:
+      required:
+        true
+  - type: textarea
+    id: goals
+    attributes:
+      label: Goals
+      description: An itemized list of goals to complete for this ticket
```

Comment:
Can we add "If you don't have specific goals, this could also include the user story or jobs to be done."

Reply:
Will swing back to address this -- need this now to help with templates

Reply:
Will cover this in my documentation update for these templates